### ISSUE TITLE
`pg` _(node-postgres)_ option missing in drizzle postgres database option

### ISSUE THREAD

**Issue by sparrowsl** [sveltejs/cli#225]
based on [this docs in drizzle](https://orm.drizzle.team/docs/get-started-postgresql) they have `pg`(node-postgres) and postgres.js.
the `pg`(node-postgres) is missing from the CLI options.

**Comment by benmccann** [comment]
Yeah, they've got tons of options, but we also want to avoid overwhelming the user. Is there a reason those ones are needed?

**Comment by sparrowsl** [comment]
the docs used these two on the getting started.

also a section on the docs states this:
> `postgres.js` uses prepared statements by default, which you may need to opt out of. **This could be a potential issue in AWS environments, among others**, so please keep that in mind.

**Comment by jycouet** [comment]
Doing a bit of triage here.

`sv` already supports 7 clients, and as mentioned in [the doc](https://svelte.dev/docs/cli/drizzle#Options-client), you can swap them according to your needs quite easily.

I’d be in favor of closing this issue until the _“most common”_ ones change.

**Comment by manuel3108** [comment]
Therefore closing
